addons/Rakugo/Rakugo.gd

```diff
@@ -86,6 +86,9 @@ func get_variable(var_name: String):
 			if store_manager.variables.has(var_name):
 				variable = store_manager.variables.get(var_name)
 			mutex.unlock()
+			
+			if variable is String:
+				return executer.replace_variables(variable)
```

Comment:
If replace_variables() is used in Rakugo.gd, move it from Executer.gd. It is better than call if from executer.
Plus user can be interested by using it.